docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`1453` Users will now see an validation error message when attempting to add an existing account.
```

Comment:
Can you also add a changelog without an issue for the ability to copy addresses from the blockchains view? This is really cool and useful.